Code/Tools/SceneAPI/SceneData/Rules/TangentsRule.cpp

```diff
@@ -143,11 +43,8 @@ namespace AZ
                     return;
                 }
 
-                serializeContext->Class<TangentsRule, DataTypes::IRule>()->Version(2)
-                    ->Field("tangentSpace", &TangentsRule::m_tangentSpace)
-                    ->Field("bitangentMethod", &TangentsRule::m_bitangentMethod)
-                    ->Field("normalize", &TangentsRule::m_normalize)
-                    ->Field("uvSetIndex", &TangentsRule::m_uvSetIndex);
+                serializeContext->Class<TangentsRule, DataTypes::IRule>()->Version(3)
+                    ->Field("tangentSpace", &TangentsRule::m_tangentSpace);
```

Comment:
This may need a version converter in case any existing data is referencing the EmotionFX tangentSpace, or is that case safeguarded somewhere else?

Reply:
The .assetinfo files are now stored as json and don't use the version converters. The json deserializer will just throw away data that is not reflected without the need to remove the elements in a version converter (they are not even called).